dotCMS/src/main/java/com/dotcms/rest/api/v1/authentication/ApiTokenResource.java

```diff
@@ -166,18 +160,26 @@ public final Response deleteApiToken(@Context final HttpServletRequest request,
     @NoCache
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-    public final Response issueApiToken(@Context final HttpServletRequest request,
-                                   @Context final HttpServletResponse response,
-                                   final ApiTokenForm formData) {
+    public final Response issueApiToken(
```

Comment:
update doc of the endpoint with new params of the form

Reply:
done https://github.com/dotCMS/core/pull/19698/commits/047eca133554e37b9db8a424b1bf434249c6826a